docs/src/InsolationExamples.md

```diff
@@ -24,16 +24,19 @@ diurnal_cycle(lat, lon, date, timezone, "Finland_June.png")
 ## Daily-mean insolation
 ### Insolation in J2000
 ```@example
-using CLIMAParameters
-using CLIMAParameters.Planet
-struct EarthParameterSet <: AbstractEarthParameterSet end
-const param_set = EarthParameterSet()
+import CLIMAParameters as CP
```

Comment:
Is CP used in the docs? Or why is this import here? (Sorry if I missed it - hard to see on my phone!)

Reply:
It is. Well, this line is actually repeated in `include(joinpath(pkgdir(Insolation), "parameters", "create_parameters.jl"))` (because it's still needed in order to create `param_set`), so we can remove this one. I'm changing `using` to `import` because it's better practice.

Reply:
I'll delete this duplicate line.